webapp/templates/jury/analysis/contest_overview.html.twig

```diff
@@ -388,7 +409,7 @@ nv.addGraph(function() {
       .showYAxis(true)        //Show the y-axis
       .showXAxis(true)        //Show the x-axis
       .forceX([0, {{ (current_contest.endtime - current_contest.starttime) / 60 }}])
-      .forceY([0, {{ submissions|length *1.10 }}])
+      .forceY([0, {{ max_submissions_per_minute * 1.10 }}])
```

Comment:
Maybe I overlook something, but where is `max_submissions_per_minute` ever updated?

Reply:
That was my oversight. I've fixed it.